qiita_db/test/test_study.py

```diff
@@ -611,8 +606,9 @@ def test_set_info(self):
         self.infoexp["spatial_series"] = None
         self.infoexp["most_recent_contact"] = None
         self.infoexp["reprocess"] = False
-        self.infoexp["lab_person_id"] = 2
         self.infoexp["first_contact"] = datetime(2014, 6, 11)
+        self.infoexp["lab_person"] = qdb.study.StudyPerson(2)
```

Comment:
Why this two step process instead of just changing the key to lab_person on line 594?